config/settings.yml

```diff
@@ -8,6 +8,7 @@ dsds:
 authentication:
   site_url: <%= ENV.fetch('AUTHENTICATION_SITE_URL') %>
   redirect_uri: <%= ENV.fetch('AUTHENTICATION_REDIRECT_URI') %>
+  token_info_uri: <%= ENV.fetch('AUTHENTICATION_TOKEN_INFO_URI') %>
```

Comment:
The token info url can be got by using the AUTHENTICATION_SITE_URL + /oauth/token/info
It won't change so no need to have as a separate config option